Apps/WebClient/src/NewClientApp/src/components/private/timeline/entry/BcCancerScreeningResultTimelineComponent.vue

```diff
@@ -0,0 +1,110 @@
+<script setup lang="ts">
+import saveAs from "file-saver";
+import { computed, ref } from "vue";
+
+import HgButtonComponent from "@/components/common/HgButtonComponent.vue";
+import MessageModalComponent from "@/components/common/MessageModalComponent.vue";
+import TimelineEntryComponent from "@/components/private/timeline/TimelineEntryComponent.vue";
+import { EntryType, entryTypeMap } from "@/constants/entryType";
+import { container } from "@/ioc/container";
+import { SERVICE_IDENTIFIER } from "@/ioc/identifier";
+import { PatientDataFile } from "@/models/patientDataResponse";
+import BcCancerScreeningResultTimelineEntry from "@/models/timeline/bcCancerScreeningResultTimelineEntry";
+import { ILogger } from "@/services/interfaces";
+import { usePatientDataStore } from "@/stores/patientData";
+import SnowPlow from "@/utility/snowPlow";
+
+interface Props {
+    hdid: string;
+    entry: BcCancerScreeningResultTimelineEntry;
+    index: number;
+    datekey: string;
+    isMobileDetails?: boolean;
+    commentsAreEnabled?: boolean;
+}
+const props = withDefaults(defineProps<Props>(), {
+    isMobileDetails: false,
+    commentsAreEnabled: false,
+});
+
+const logger = container.get<ILogger>(SERVICE_IDENTIFIER.Logger);
+const patientDataStore = usePatientDataStore();
+const messageModal = ref<InstanceType<typeof MessageModalComponent>>();
+
+const entryIcon = computed(
+    () => entryTypeMap.get(EntryType.BcCancerScreening)?.icon
+);
+const isLoadingFile = computed(
+    () =>
+        props.entry.fileId !== undefined &&
+        patientDataStore.isPatientDataFileLoading(props.entry.fileId)
+);
+const hasFile = computed(
+    () => props.entry.fileId !== undefined && props.entry.fileId !== ""
+);
+
+function showConfirmationModal(): void {
+    messageModal.value?.showModal();
+}
+
+function downloadFile(): void {
+    if (props.entry.fileId) {
+        SnowPlow.trackEvent({
+            action: "download_report",
+            text: "BC Cancer Result PDF",
+        });
+        const dateString = props.entry.date.format("yyyy_MM_dd-HH_mm");
+        patientDataStore
+            .retrievePatientDataFile(props.hdid, props.entry.fileId)
+            .then(
+                (patientFile: PatientDataFile) =>
+                    new Blob([new Uint8Array(patientFile.content)], {
+                        type: patientFile.contentType,
+                    })
+            )
+            .then((blob) => saveAs(blob, `cancer_screening_${dateString}.pdf`))
+            .catch((err) => logger.error(err));
+    }
+}
+</script>
+
+<template>
+    <TimelineEntryComponent
+        :card-id="`${index}-${datekey}`"
+        :entry-icon="entryIcon"
+        icon-class="bg-primary"
+        :title="entry.title"
+        :subtitle="`Program: ${entry.programName}`"
```

Comment:
Is the "Program: " prefix required?  It's not listed in the story requirements.

Reply:
its in the designs

Reply:
I'd recommend double-checking with Nino/Mandad to be sure.